src/frontend/details/details.css

```diff
@@ -11,6 +11,10 @@
     justify-content: center
 }
 .details .content .cover img{
-    max-width:  60%;
+    max-width:  58%;
     border: 8px solid white;
+    background-color: rgb(245, 245, 245);
+    -webkit-box-shadow: 0 15px 20px -8px rgba(0, 0, 0, 0.2);
```

Comment:
You don't need these vendor prefixes any more, modern browsers support shadows already :)